### DISCUSSION TITLE
Json to std::unordered_map

### DISCUSSION THREAD

**Discussion by mPlauts** [nlohmann/json#3689] [Q&A] [answered]
Hello. I need my json to look like this
{ 
  "Staff List": { 
  "Staff": ["NameHere" , 29 ]
  } 
}
i can compose json using string
m_data = ("{ \"Staff List\": { \"Staff\": [\""+ NameHere+"\" , " + std::to_string(age) + " ] } }");
but I can't convert the json back to read the value. I would like to store this data in std::unordered_map<std::string, std::uint8_t>

**Comment by falbrechtskirchinger** [comment, marked as answer]
```cpp
auto j = json::parse(R"("{ "Staff List": { "Staff": ["NameHere" , 29 ] } }")");
```

You can iterate through it using https://json.nlohmann.me/api/basic_json/items/ and populate a `std::unordered_map`.
